### DISCUSSION TITLE
Satellite Map Layer

### DISCUSSION THREAD

**Discussion by Suburbanno** [organicmaps/organicmaps#381] [Ideas]
Add satellite view layer option

**Comment by FinixFighter** [comment]
A street view feature (similar to that of OsmAnd) would be great too! Images from Bing, KartaView and Mapillary could be used.

**Reply by biodranik** [reply]
It won't work in offline.

**Reply by FinixFighter** [reply]
> It won't work in offline.

It could be an "online only" feature. If you enable the street view but you're offline, you'll get an error message saying that that specific feature is not available offline. OsmAnd is an offline map too, but it has street view.

**Comment by Phantom63rus** [comment]
Maxar, ESRI...

Strava, may be Open Weather.

Yes, online only, it's normal. For offline navigation OSM layer.

**Comment by PinguDEV-original** [comment]
Well maybe SentinelMap, CC-BY-SA 4.0, idk hoe big it is, but shouldn't be soo big. Would be really awesome to have

**Comment by pastk** [comment]
Tracked in https://github.com/organicmaps/organicmaps/issues/618
